src/libraries/System.Reflection.Emit/tests/AssemblyBuilderTests.cs

```diff
@@ -424,7 +424,16 @@ void Invoke_Private_SameAssembly_ThrowsMethodAccessException()
 	    Assert.Throws<MethodAccessException>(() => d ());
 	}
 
+    [Fact]
+    public void DefineDynamicAssembly_InternalAssemblyLocationIsEmpty()
+    {
+        AssemblyBuilder assembly = Helpers.DynamicAssembly();
```

Comment:
```suggestion
        AssemblyBuilder assembly = Helpers.DynamicAssembly(nameof(DefineDynamicAssembly_InternalAssemblyLocationIsEmpty));
```

Otherwise the assembly will get a default name `TestAssembly` and the test might succeed by accident if some other test uses the same assembly name. (Unlikely as the test would still likely do the right validation, but still)